openmc/weight_windows.py

```diff
@@ -441,7 +445,7 @@ def from_hdf5(cls, group, meshes):
         )
 
 
-def wwinp_to_wws(path):
+def wwinp_to_wws(path) -> WeightWindows:
```

Comment:
This actually returns a list of weight windows:
```suggestion
def wwinp_to_wws(path) -> List[WeightWindows]:
```